package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@loke/http-kit",
-  "version": "1.0.3",
+  "version": "1.1.0",
```

Comment:
Is this the right level of update?
I'm not sure what the standard is.
My instinct is that version 1.1 would have different functionality from 1.0.

Reply:
[source](https://docs.npmjs.com/about-semantic-versioning#incrementing-semantic-versions-in-published-packages)

<img width="675" alt="image" src="https://user-images.githubusercontent.com/7439294/227847503-3c3a9c1a-1119-4e77-bc7f-01eb8df50c0f.png">

Reply:
Seems to be non-breaking so not a major bump.

Just depends on whether you view supporting a wider set of prom-client versions as a feature or a bug fix.